Emulator/pal/gc.asm

```diff
@@ -44,7 +42,7 @@ TTYO,	0						/TTY OUTPUT SUB-ROUTINE.
 		CLA						/CLEAR ACC
 		JMP I TTYO				/RETURN 
 
-MSG,	.						/POINTER TO STRING
+MSG,							/POINTER TO STRING
```

Comment:
This was actually a bug. The period "." means address where current line will end up in memory, so in effect the first character in your string is the address of the string.  DOH! I see now that you use AIX so the first character will be skipped... ignore this line :D I wonder if github makes it easy to remove a single line from a pull request.

Anyway. the period is mostly useful to write things like ".+2" which then means the current address +2 locations. It seems popular to write .-. to get zero.

Reply:
Reverted.